scarf/writers.py

```diff
@@ -1109,264 +1110,6 @@ def dump(self):
                     s = e
 
 
-class ZarrMerge:
```

Comment:
we still need to keep ZarrMerge in name space for backward compatibility with deprecation warning